snuba/query/snql/expression_visitor.py

```diff
@@ -66,7 +69,11 @@ def get_arithmetic_expression(
         return get_arithmetic_function(exp.op)(term, exp.arithm, None)
     elif isinstance(exp, list):
         for elem in exp:
-            term = get_arithmetic_function(elem.op)(term, elem.arithm, None)
+            if isinstance(elem, (LowPriTuple, HighPriTuple)):
+                term = get_arithmetic_function(elem.op)(term, elem.arithm, None)
+            elif isinstance(elem, list):
```

Comment:
Can the elements themselves be lists? From the signature it seems they can only be HiPriTuple or LowPriTuple

Reply:
I changed the signature, but yes it can be an array of alternating space / tuple, e.g. (`[Node, LowPriTuple, Node, LowPriTuple]`).